pootle/settings/40-apps.conf

```diff
@@ -131,6 +131,23 @@ POOTLE_SIGNUP_ENABLED = True
 # chart with events
 POOTLE_REPORTS_MARK_FUNC = ''
 
+# Invoicing
+
+# Path to PhantomJS binary (needed to generate PDF invoices)
+POOTLE_INVOICES_PHANTOMJS_BIN = None
+
+# Base directory where PDF invoices will be created
+POOTLE_INVOICES_DIRECTORY = working_path('invoices')
```

Comment:
The configured directory will be created if missing, provided the command has enough permissions to do so: I believe it's safe to assume so as this should be the case if `working_path('invoices')` wants to be used.